src/pages/Trainee/TraineeList.jsx

```diff
@@ -148,10 +159,29 @@ class TraineeList extends React.Component {
   };
 
   componentDidMount = () => {
-    this.setState({ loading: true });
+    this.setState({ loading: false });
     this. getTraineeData(); 
   }
    getTraineeData= async()=>{
+    // eslint-disable-next-line consistent-return
+    // callApi({ }, 'get', `/trainee?skip=${0}&limit=${10}`).then((res) => {
+    //   this.setState({ dataObj: res.data.data.records, loading: false, Count: 100 });
+    //   console.log('getresponse', res);
+    //   console.log('recordlength', res.data.data.records.length)
+
+    //   if (res.data.status !== 200) {
+    //     this.setState({
+    //       loading: false,
+    //       Count: 100,
+
+    //     }, () => {
+    //       console.log('call Api');
+    //     });
+    //   } else {
+    //     this.setState({ dataObj: trainees, loading: false, Count: 100 });
+    //     return res;
+    //   }
+    // });
      const { page, rowsPerPage } = this.state;
```

Comment:
Since we have added qraphql query to fetch data from graphql server, No need to call API directly. remove this